packages/terra-framework-docs/src/terra-dev-site/test/worklist-data-grid/WorklistDataGridWithRowSelection.test.jsx

```diff
@@ -38,6 +38,18 @@ const WorklistDataGridWithRowSelection = () => {
     clearRowSelection();
   };
 
+  const onRowSelect = (rowIdsToSelect, rowIdsToUnselect) => {
```

Comment:
I'm noticing we're doing this quite a lot repeatedly. Is there possibly a way we could specify this as a util function and then just repeat that everywhere we need this?